fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/FabricDataGenerator.java

```diff
@@ -102,29 +102,45 @@ public DataGenerator.Pack createVanillaSubPack(boolean shouldRun, String packNam
 		throw new UnsupportedOperationException();
 	}
 
+	/**
+	 * Represents a pack of generated data (i.e. resource pack). Providers are added to a pack.
```

Comment:
```suggestion
	 * Represents a pack of generated data (i.e. data pack or resource pack). Providers are added to a pack.
```

It's not always a resource pack

Reply:
"Resource pack" in modding world usually refers to `ResourcePack`, but yeah